haskell/platforms/BUILD

```diff
@@ -0,0 +1,10 @@
+# This file declares constraint values for each platform supported by
+# GHC. These rules follow the GHC naming convention, for example,
```

Comment:
Please add a link to GHC naming convention.

Reply:
This is not a GHC specific convention, nor is canonical documentation available. I added a link to `config.guess`, which is essentially standard autoconf stuff.